test/framework/multicluster.go

```diff
@@ -119,9 +119,17 @@ func (m *MulticlusterE2ETest) CreateManagementCluster(opts ...CommandOpt) {
 	m.ManagementCluster.CreateCluster(opts...)
 }
 
+// CreateTinkerbellManagementClusterWithConfig first generates a cluster config based on the management cluster test's
+// previous configuration and proceeds to run tinkerbell related steps for cluster creation.
+func (m *MulticlusterE2ETest) CreateTinkerbellManagementClusterWithConfig(opts ...CommandOpt) {
+	m.ManagementCluster.GenerateClusterConfig()
```

Comment:
Can we use `WithClusterConfig` instead of `GenerateClusterConfig` to construct the initial cluster config?
 
The reason being we want out tests to not relying on storing the yaml content of the manifest and just use a just use a cluster.Config. See the full reasoning [here](https://github.com/aws/eks-anywhere/pull/4316)

Reply:
Updating, and will test some of the affected tests